internal/server/events_test.go

```
@@ -15,11 +15,11 @@ func TestSSEConnection(t *testing.T) {
 	testFile := filepath.Join(dir, "test.md")
 	os.WriteFile(testFile, []byte("# Test"), 0o644)
 
-	hub := NewSSEHub(dir, nil, nil)
+	hub := NewEventHub(dir, nil, nil)
 	hub.Start()
```

golangci-lint (2.11.4)

[error] Error return value of `hub.Start` is not checked

(errcheck)